### DISCUSSION TITLE
Which font is used in the demo video?

### DISCUSSION THREAD

**Discussion by HumblePresent** [Pocco81/TrueZen.nvim#95] [Q&A] [answered]
Question in the title! Very cool plugin btw. Plan on trying it out tonight!

**Comment by pocco81** [comment, marked as answer]
Sorry for the late reply, it's `Iosevka Medium Nerd Font Complete`, you can get it from [here](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Iosevka).

Hope it helps! :)
